pkg/vsphere/session/session.go

```diff
@@ -202,6 +204,7 @@ func (s *Session) Connect(ctx context.Context) (*Session, error) {
 	}
 
 	soapClient := soap.NewClient(soapURL, s.Insecure)
+
```

Comment:
Minor: unneeded whitespace.

Reply:
I'm going to ignore this to avoid further churn in this file.